addons/kotsadm/alpha/install.sh

```diff
@@ -81,17 +82,24 @@ function kotsadm_outro() {
     printf "\n"
 }
 
-function kotsadm_secret_auto_create_cluster_token() {
-    local AUTO_CREATE_CLUSTER_TOKEN=$(kubernetes_secret_value default kotsadm-auto-create-cluster-token token)
+function kotsadm_secret_cluster_token() {
+    local CLUSTER_TOKEN=$(kubernetes_secret_value default kotsadm-cluster-token token)
```

Comment:
the field that it's stored under is `kotsadm-cluster-token` too

Reply:
fixed